### PR TITLE
Support set options to update vote tx by lua script

### PR DESCRIPTION
**Pull request by peterwillcn**

### CODE REVIEW

File: test/transaction/vote_cr_tx.lua [hunk 1]
```diff
@@ -43,24 +41,22 @@ then
...
-if vote_candidate_vote == 0
+if vote_candidates == ""
 then
-    vote_candidate_vote = 0.1
+    print("candidates is nil, should use --candidates or -cds to set it.")
+    return
 end
 
-if vote_candidate_code == ""
+if vote_candidate_votes == ""
```
Comment (Houshoupei84) [inline test/transaction/vote_cr_tx.lua]: should be {}

File: test/transaction/vote_cr_tx.lua [hunk 2]
```diff
@@ -43,24 +41,22 @@ then
     fee = 0.1
 end
 
-if vote_candidate_vote == 0
+if vote_candidates == ""
```
Comment (Houshoupei84) [inline test/transaction/vote_cr_tx.lua]: should be {}

File: test/transaction/vote_tx.lua [hunk 3]
```diff
@@ -42,24 +40,22 @@ then
 	fee = 0.001
 end
 
-if vote_candidate_vote == 0
+if vote_candidates == ""
```
Comment (Houshoupei84) [inline test/transaction/vote_tx.lua]: same as above

File: test/transaction/vote_cr_tx.lua (hunk 1 above)
Comment (peterwillcn) [reply test/transaction/vote_cr_tx.lua]: should be if next(vote_candidates) == nil

File: test/transaction/vote_cr_tx.lua (hunk 2 above)
Comment (peterwillcn) [reply test/transaction/vote_cr_tx.lua]: should be if next(vote_candidates) == nil

**Pull request merged**